### PR TITLE
Remove bugged tooltip and redraw table on sidenav collapse

### PR DESCRIPTION
**Pull request by Christoffer-Cortes**

### CODE REVIEW

File: src/main/resources/static/js/main.js [hunk 1]
```diff
@@ -17,36 +17,43 @@ jQuery(document).ready(function() {
 	var frontendServiceUrl = $('#frontendServiceUrl').text();
 
 	function loadMainPage() {
+		$("#navbarResponsive").removeClass("show");
```
Comment (a-breid-ericsson) [inline src/main/resources/static/js/main.js]: Why is the indention here ...
```
Line one
        Line 2
    Line 3
        Line4
```
?

Comment (Christoffer-Cortes) [reply src/main/resources/static/js/main.js]: It's caused by the use of tab in the source code... it hasn't been formatted correctly.

File: src/main/resources/static/js/subscription.js [hunk 2]
```diff
@@ -414,6 +414,10 @@ jQuery(document).ready(function () {
             }
         }
     });
+
+    $("#sidenavCollapse").click(function() {
+		table.ajax.reload();
```
Comment (tobiasake) [inline src/main/resources/static/js/subscription.js]: Does this mean that Datatable is reloaded and it re-retrieve data from backend, e.g. it makes a Get all subscriptions request everytime sidebar is collapsed?

Comment (Christoffer-Cortes) [reply src/main/resources/static/js/subscription.js]: Apparently the function was unable to detect dataset changes but I've now changed it with a better method.

**Pull request merged**